TypeFinder.py

```diff
@@ -392,38 +450,6 @@ def typing_kit(file_name) :
 
 
     ##===============
-    #This imports Cruz et al. 2017 templates into astropy tables
-    ##===============
-
-    #Beginning with the field templates. J-H-K bands are all diff lengths and so need their own table
-    field_templates = defaultdict(dict)
-
-    for band in ['J', 'H', 'K']:
-        for spectype in range(9):
-            band_data = ascii.read("templates/L{}{}_f.txt".format(spectype, band))
-            field_templates[band][spectype] = Table([band_data['col1'], band_data['col2'], band_data['col4'], band_data['col5']], \
-                                                    names = ('wavelength', 'flux', 'upper', 'lower'))
-
-    #Next with beta templates
-    beta_templates = defaultdict(dict)
-
-    for band in ['J', 'H', 'K']:
-        for spectype in range(2):
-            band_data = ascii.read("templates/L{}{}_b.txt".format(spectype, band))
-            beta_templates[band][spectype] = Table([band_data['col1'], band_data['col2'], band_data['col4'], band_data['col5']], \
-                                                   names = ('wavelength', 'flux', 'upper', 'lower'))
-
-    #Next with gamma templates
-    gamma_templates = defaultdict(dict)
-
-    for band in ['J', 'H', 'K']:
-        for spectype in range(5):
-            band_data = ascii.read("templates/L{}{}_g.txt".format(spectype, band))
-            gamma_templates[band][spectype] = Table([band_data['col1'], band_data['col2'], band_data['col4'], band_data['col5']], \
-                                                    names = ('wavelength', 'flux', 'upper', 'lower'))
-    
-
-    ##===============
     #This defines the initial grid plot
     ##===============
     
```

Comment:
If the information between the "header" and "footer" was deleted so should the header and footer.